AUTH-README.md

```diff
@@ -0,0 +1,61 @@
+# Authentication
+
+As outlined in <a href="https://github.com/grpc/grpc-common/blob/master/grpc-auth-support.md">gRPC Authentication Support</a>, gRPC supports a number of different mechanisms for asserting identity between an client and server. This document provides code samples demonstrating how to provide SSL/TLS support encryption and identity assertions in Java, as well as passing OAuth2 tokens to services that support it.
+
+# Java 7, HTTP2 & Crypto
+
+## Cipher-Suites
+Java 7 does not support the <a href="https://tools.ietf.org/html/draft-ietf-httpbis-http2-17#section-9.2.2">the cipher suites recommended</a> by the HTTP2 specification. To address this we suggest servers use Java 8 where possible. If this is not possible it is possible to use other ciphers but you need to ensure that the services you intend to call have <a href="https://github.com/grpc/grpc/issues/681">allowed out-of-spec ciphers</a>. On Android we recommend the use of the <a href="http://appfoundry.be/blog/2014/11/18/Google-Play-Services-Dynamic-Security-Provider/">Play Services Dynamic Security Provider</a> to ensure your application has an up-to-date OpenSSL library with the necessary ciper-suites and a reliable ALPN implementation.
```

Comment:
I assume we're mostly talking about bouncycastle here. Looks like it supports
TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
and other allowed cousins. Adding a note.

FYI we're looking at using Netty's openssl bindings to get around this using JNI rather than Xbootclasspath which is ironically easier for users to consume. In theory Okhttp could use this too for non-Android environments. WDYT?